src/common/symbols.map

```diff
@@ -691,8 +696,12 @@ global:
 };
 
 MIR_COMMON_2.18 {
-  global: extern "C++" {
+global:
+  extern "C++" {
     MirTouchpadConfig::disable_with_external_mouse*;
     mir::event_type_to_c_str*;
+    std::hash::operator*;
+    typeinfo?for?std::hash;
```

Comment:
We're only providing a `hash` specialization on `mir::SharedLibrary::Handle`, we shouldn't be exporting arbitrary other instantiations (they are hard to control/check for changes).